tools/rotate-certs.sh

```diff
@@ -37,3 +37,7 @@ ${_kubectl} delete pods --namespace "${namespace}" -l cdi.kubevirt.io
 echo "# Rotating kubevirt certificates ..."
 ${_kubectl} delete secrets --namespace "${namespace}" -l kubevirt.io
 ${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io
+
+echo "# Rotating SSP certificates ..."
+${_kubectl} delete secrets --ignore-not-found=true --namespace "${namespace}" virt-template-validator-certs
+${_kubectl} delete pods --namespace "${namespace}" -l kubevirt.io=virt-template-validator
```

Comment:
I am not sure the validator regenerates them on first start, better check that with @fromanirh